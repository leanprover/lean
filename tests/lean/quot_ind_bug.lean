open quot

variables {A : Type} [s : setoid A] {B : quot s → Prop} (c : ∀ (a : A), B (quot.mk a)) (a : A)
check quot.ind c ⟦a⟧
check c a
eval quot.ind c ⟦a⟧
