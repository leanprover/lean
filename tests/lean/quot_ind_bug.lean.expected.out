ind c ⟦a⟧ : B ⟦a⟧
c a : B ⟦a⟧
c a
